.github/dependabot.yml

```diff
@@ -0,0 +1,106 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: docker
+    directory: /3rd-party/tor
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /packages/backend-bundle
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: docker
+    directory: /packages/backend
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /packages/backend
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /packages/common
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /packages/desktop
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: docker
+    directory: /packages/e2e-tests/docker
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /packages/e2e-tests
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /packages/eslint-config-custom
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /packages/identity
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: docker
+    directory: /packages/integration-tests
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /packages/integration-tests
```

Comment:
I would not check integration-tests right now, we have to clean up the package first. I believe that it was not adjusted after we introduced nest to the project. I am afraid that it would be messy if dependabot checked it right now but maybe I am wrong

Reply:
Resolved in c9387db